lib/rtp_h264/plugin_app.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Membrane.RTP.H264.Plugin.App do
+  @moduledoc false
+  use Application
+  alias Membrane.RTP.{H264, PayloadFormat}
+
+  def start(_type, _args) do
+    PayloadFormat.register(%PayloadFormat{
```

Comment:
I think I haven't seen that in core, but looks cool! :smile:

Reply:
This is a part of membrane_rtp_format